### ISSUE TITLE
`tmle3mopttx` with missing baseline

### ISSUE THREAD

**Issue by nhejazi** [tlverse/tlverse-handbook#13]
It seems that the version of `tmle3mopttx` tagged in the `DESCRIPTION` file (specifically, [this version](https://github.com/tlverse/tmle3mopttx/tree/5ba5f65c0de0835aae48db5968aa11ae918cc000)) doesn't fully support working with missing data, as identified in [this Travis build](https://travis-ci.com/github/tlverse/tlverse-handbook/builds/220023790#L2367). This only affects the last example given in the chapter, which is turned off in [this PR](https://github.com/tlverse/tlverse-handbook/commit/20f39fdbfa4cd91c02c62a38955d370e2020762e#diff-35b3d6a777db4195141c18d342f736bfd665790755c978927d80722cc304e6e4R961) in order to restore the automated builds. Not urgent to fix, but just recording here for bookkeeping.

**Comment by jbriggs7** [comment]
Hi, I am also encountering this issue when I run the exercise in the chapter: https://tlverse.org/tlverse-handbook/optimal-individualized-treatment-regimes.html. I get the same error which is that W has censoring and it is not supported.

**Comment by nhejazi** [comment]
Thanks for reporting this @jbriggs7. The issue should be resolved in using the latest version of the `tmle3mopttx` package (installable via `devtools::install_github("tlverse/tmle3mopttx")`). Also @podTockom notes that the exercise has been updated so that missing values in baseline covariates are no longer considered (even though this should no longer be an issue when using the latest version of the package).

We've also updated the book to use this latest version of `tmle3mopttx` automatically, which it was not doing before. I'll close this issue but feel free to report if there are any other issues that come up.
